### PR TITLE
Jira Table Links

### LINKED ISSUE #177: Jira Table Link Titles
### Feature
Update the title component to a link which points to the query used to form the table.
`<Link to={'${getJiraUrl(project)}issues/?jql=${tableContent.query}'}>`
image

### Value
Jira Dashboard users would have easy access to the filter within JIRA.

### Context
Current the searchJira, getIssuesByComponent.. functions in [api.ts](https://github.com/AxisCommunications/backstage-plugins/blob/5a933c34eacedb35e940ec5ff10a73254ea1b523/plugins/jira-dashboard-backend/src/api.ts) all return response.issues rather than the full response object which contains the JQL query. The return value will need to be changed to include the query.

### PR DESCRIPTION
**Pull request by Aaron-Wood-Eric**
### Context

Implementation for links within the Jira Table.

image

The JiraDataResponse was updated to add the jql query where appropriate.

### Issue ticket number and link

- Fixes #177 

### Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have verified that the code builds perfectly fine on my local system
- [x] I have verified that my code follows the style already available in the repository
- [x] A changeset describing the change and affected packages. ([more info](https://github.com/AxisCommunications/backstage-plugins/blob/main/CONTRIBUTING.md#changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by fridajac** [comment]
Nice work! I've added a few comments.

**Comment by Aaron-Wood-Eric** [comment]
> Nice work! I've added a few comments.

@fridajac There seems to be no comments visible :)

File: plugins/jira-dashboard/src/components/JiraTable/JiraTable.tsx [hunk 1]
```diff
@@ -66,12 +70,25 @@ export const JiraTable = ({
...
       {`${capitalize(tableContent.name)} (${nbrOfIssues})`}
     </Typography>
   );
 
+  if (project && tableContent.query) {
+    title = (
+      <Link
+        to={`${getJiraBaseUrl(project.self)}/issues/?jql=${tableContent.query}`}
+        variant="h5"
+        component="div"
```
[the remark this answers is not in the document]
Comment (Aaron-Wood-Eric) [reply plugins/jira-dashboard/src/components/JiraTable/JiraTable.tsx]: Fixed

**Review by fridajac — APPROVED** [review]
Nice work! Thank you for your contribution. We appreciate it :100:

**Pull request merged**